app/dashboard/templates/onepager/send2.html

```diff
@@ -80,18 +80,24 @@ <h1>{% trans "Send Tip." %}</h1>
     <div>
       <br>
       {% trans "Amount of" %}
-      <select id="token" style="width:100px; margin-bottom: 10px; display: inline;" >
-        <option value="0x0">ETH</option>
-      </select>
-      <input name="amount" type="text" placeholder="1.1" id="amount" value="" style="width: 170px; display: inline;">
+        {% if fund_request.network == 'ETH' %}
+        <select id="token" style="width:100px; margin-bottom: 10px; display: inline;" data-token="{{ fund_request.token_name }}">
+          <option value="0x0">ETH</option>
+        </select>
+        {% else %}
+        <select style="width:100px; margin-bottom: 10px; display: inline;">
+            <option value="{{ fund_request.network }}">{{ fund_request.network }}</option>
+        </select>
```

Comment:
Hey @zoek1 just to let you know here was missing the id="token" so the token select wasn't getting fill.

Reply:
Sorry @octavioamu, I'm not sure why the id was missing. I really tested :anguished: